packages/roosterjs-editor-api/lib/format/cacheGetListElement.ts

```diff
@@ -5,19 +5,26 @@ import { getTagOfNode } from 'roosterjs-editor-dom';
 
 const EVENTDATACACHE_LISTELEMENT = 'LISTELEMENT';
 
-export default function cacheGetListElement(editor: Editor, event?: PluginEvent): Element {
+export default function cacheGetListElement(editor: Editor, event?: PluginEvent, listType?: string): Element {
```

Comment:
Suggest to use string literal type for listType. For example,
type ListType = 'LI' | 'BLOCKQUOTE';

Reply:
Changed